Sources/SFSafeSymbols/Symbols/SFSymbol+3.3.swift

```diff
@@ -0,0 +1,47 @@
+// Don't touch this manually, this code is generated by the SymbolsGenerator helper tool
+
+// 3.3 Symbols
+@available(iOS 15.4, macOS 12.3, tvOS 15.4, watchOS 8.5, *)
+public extension SFSymbol {
+    /// No preview available
+    /// Single Localization, ? Layersets
+    ///
+    /// Unavailable layerset information
+    static let cameraMacro = SFSymbol(rawValue: "camera.macro")
+
+    /// No preview available
+    /// Single Localization, ? Layersets
+    ///
+    /// Unavailable layerset information
+    static let cameraMacroCircle = SFSymbol(rawValue: "camera.macro.circle")
+
+    /// No preview available
+    /// Single Localization, ? Layersets
+    ///
+    /// Unavailable layerset information
+    static let cameraMacroCircleFill = SFSymbol(rawValue: "camera.macro.circle.fill")
+
+    /// No preview available
+    /// Single Localization, ? Layersets
+    ///
+    /// Unavailable layerset information
+    static let dotsAndLineVerticalAndCursorarrowRectangle = SFSymbol(rawValue: "dots.and.line.vertical.and.cursorarrow.rectangle")
+
+    /// No preview available
+    /// Single Localization, ? Layersets
+    ///
+    /// Unavailable layerset information
+    static let keyViewfinder = SFSymbol(rawValue: "key.viewfinder")
+
+    /// No preview available
+    /// Single Localization, ? Layersets
+    ///
+    /// Unavailable layerset information
+    static let personBadgeKey = SFSymbol(rawValue: "person.badge.key")
+
+    /// No preview available
+    /// Single Localization, ? Layersets
+    ///
+    /// Unavailable layerset information
+    static let personBadgeKeyFill = SFSymbol(rawValue: "person.badge.key.fill")
```

Comment:
Optional: I would rather write `Layerset information unavailable`, but that's a matter of preference, so decide how you want